crates/primitives/src/explorer.rs

```diff
@@ -209,6 +209,18 @@ impl<'a> Explorer<'a> {
                 account_url: "/address/",
                 token_url: None,
             },
+            Chain::Dymension => Explorer {
+                host: "https://www.mintscan.io/dymension",
+                transaction_url: "/tx/",
+                account_url: "/address/",
+                token_url: Some("/assets/"),
+            },
+            Chain::Saga => Explorer {
+                host: "https://www.mintscan.io/saga",
+                transaction_url: "/tx/",
+                account_url: "/address/",
```

Comment:
does token url works? I think it should be `None`

Reply:
nice catch, saga doesn't have tokens yet, we can use `None` for now